src/brushes/PencilBrush.ts

```diff
@@ -2,17 +2,17 @@ import { ModifierKey, TEvent } from '../EventTypeDefs';
 import { Point } from '../Point';
 import { Shadow } from '../Shadow';
 import { Path } from '../shapes/Path';
-import { PathData } from '../typedefs';
-import { getSmoothPathFromPoints, joinPath } from '../util/path';
+import { getSmoothPathFromPoints, joinPath } from '../util/path/path';
 import type { Canvas } from '../canvas/Canvas';
 import { BaseBrush } from './BaseBrush';
+import { TSimplePathData } from '../util/path/path_types';
```

Comment:
i think the new pattern is to use camelCase for files.
I say i think because it wasn't really planned it just happened.
What about `../util/path/typedefs` ?

Reply:
done